.git-blame-ignore-revs

```diff
@@ -0,0 +1,19 @@
+# List of commits that `git blame` should ignore.
+#
+# Include ONLY commits that are known to be no-ops, like automated
+# reformattings.
+#
+# To properly format an entry, run
+#
+#     git show --no-patch --format='%H  # %s' COMMIT_HASH
+#
+# replacing `COMMIT_HASH` with the appropriate hash, or pass similar
+# formatting to `git log`, etc.
+#
+# To teach your Git client to read this file, run
+#
+#     git config blame.ignoreRevsFile .git-blame-ignore-revs
+#
+# (with Git 2.23+). See `git help blame` `git help config` for details.
```

Comment:
Nit: suggest rephrasing / adding it's on paragraph to make clear this means a version requirement and it's not at all supported in earlier versions.

Reply:
Done.